src/containers/navigation/navigationItemFieldsFragment.gql

```diff
@@ -0,0 +1,11 @@
+fragment NavigationItemFields on NavigationItemData {
+  content {
+    language
+    value
+  }
```

Comment:
`content` is no longer needed, right? It could be a lot more data so we shouldn't request it if we don't need it.